### PR TITLE
New chem- autisticium

### PR DESCRIPTION
**Pull request by Anderbergst**

### CODE REVIEW

**Comment by Anderbergst** [comment]
my first time coding someone help reeeeeeeeee

**Comment by TheOneAndOnlyCreeperJoe** [comment]
okay

this is fine

**Comment by Anderbergst** [comment]
is there anything wrong with the code?

**Comment by yobob591** [comment]
fucking best chem ever +1

**Comment by Anderbergst** [comment]
please help me with this code, or atleast point out errors and ways to improve it etc.

**Comment by ArcLumin** [comment]
uh

results in 100u

instagib in less than 1u

make, water down with 900u water

have 1000u of instakill juice

spray people with extinguisher of it

**Comment by Anderbergst** [comment]
you literally need 100 chemicals to make it

**Comment by yobob591** [comment]
you cant hope to make this in the length of a hippie round

**Comment by ArcLumin** [comment]
you underestimate the autism of the crew

**Comment by ghost** [comment]
You have to go back

**Comment by Anderbergst** [comment]
In my estimate it would take 1 hour and 30 minutes to make this

**Comment by Anderbergst** [comment]
if you are fast

**Comment by Anderbergst** [comment]
and no one fucks with you

**Comment by ArcLumin** [comment]
if this gets added I will prove you wrong and do it

**Comment by yobob591** [comment]
this is more of an admin meme chem than anything

**Comment by yobob591** [comment]
for admins to add 10000000000 units to everyone :^)

**Comment by Anderbergst** [comment]
i mean the crew can make it if they are autistic enough

File: Autisticium [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/datum/chemical_reaction/autisticium
```
Comment (JohnGinnane) [inline Autisticium:1]: This should really be in the chemistry reagents or something

File: Autisticium [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	result_amount = 100
+  required_temp = 1000
```
Comment (JohnGinnane) [inline Autisticium]: Indentation is wrong, needs to be inline with the previous line

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

File: Autisticium (hunk 1 above)
Comment (Anderbergst) [reply Autisticium:1]: okay this is my first time coding so im gonna need so much help to make this work. Okay thanks

File: Autisticium (hunk 2 above)
Comment (Anderbergst) [reply Autisticium]: got it. thanks

File: Autisticium (hunk 1 above)
Comment (Anderbergst) [reply Autisticium:1]: uhh can you teach me how to edit the code i cant seem to edit it

Comment (Anderbergst) [reply Autisticium:1]: nvm i found it

**Comment by JohnGinnane** [comment]
Right well usually chemical reactions like this are stored in something like https://github.com/HippieStationCode/HippieStation13/blob/56a3fdddeb58b6e0dedb20514f4cc43bee628c58/code/modules/reagents/Chemistry-Recipes/Others.dm

**Comment by JohnGinnane** [comment]
Updated above to be correct.

The chemicals themselves are stored here (at least, misc chemicals like spraytan are):

https://github.com/HippieStationCode/HippieStation13/blob/1feb23bce65fd8ba32f06d622b0c5836b9506d20/code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

Here's where you'll define the chemical you can create (autisticium) and how it works (the on_mob_life() proc)

**Comment by Anderbergst** [comment]
okay thansk

**Comment by Anderbergst** [comment]
Okay I fixed everything like you said.

Any other suggestions or fix?

**Comment by JohnGinnane** [comment]
See my above comments about where the code should actually be, have a look at other chemicals in those files (e.g. blood, or water) and see how they're done.

**Comment by Anderbergst** [comment]
okay

**Comment by Anderbergst** [comment]
Oh so i have to make a new line of code?

**Comment by JohnGinnane** [comment]
You'll have to put your code into two separate files, one for the "reaction" (when you mix all those ingredients together), and one for the chemical itself (where you'll also have your "on_mob_life" proc)

**Comment by Kierany9** [comment]
nigger what the fuck are you doing?

**Comment by yobob591** [comment]
merge

**Comment by Kierany9** [comment]
no seriously

**Comment by Anderbergst** [comment]
How do you put the code into two seperate files?

Im a total newbie so please help.

**Comment by JohnGinnane** [comment]
Did you follow the guide to set up GitHub?

**Comment by Anderbergst** [comment]
uhh can you link me the guide? im new to github too ;/

**Comment by JohnGinnane** [comment]
http://wiki.hippiestation.com/index.php?title=Guide_to_contributing_to_the_game

**Comment by Anderbergst** [comment]
oh i have to download stuff and etc. This will take awhile.

**Comment by Anderbergst** [comment]
okay

**Comment by JohnGinnane** [comment]
Good luck! :¬)

**Pull request closed without merging**